content/it/docs/reference/glossary/container-runtime.md

```diff
@@ -2,7 +2,7 @@
 title: Container Runtime
 id: container-runtime
 date: 2019-06-05
-full_link: /docs/reference/generated/container-runtime
+full_link: /docs/setup/production-environment/container-runtimes
```

Comment:
`full_link: /docs/setup/production-environment/container-runtimes`
vs.
`full_link: /it/docs/setup/production-environment/container-runtimes`,

which one would be better? 😊
(It seems that both one works and does not affect the UX.)

FYI: There exists inconsistency among existing localized files.

- [ko](https://raw.githubusercontent.com/kubernetes/website/master/content/ko/docs/reference/glossary/container-runtime.md): `full_link: /docs/setup/production-environment/container-runtimes`
- [zh](https://raw.githubusercontent.com/kubernetes/website/master/content/zh/docs/reference/glossary/container-runtime.md): `full_link: /zh/docs/setup/production-environment/container-runtimes`
- [ja](https://raw.githubusercontent.com/kubernetes/website/master/content/ja/docs/reference/glossary/container-runtime.md): `full_link: /ja/docs/setup/production-environment/container-runtimes`